accounts/erd1qqqqqqqqqqqqqpgq8xwzu82v8ex3h4ayl5lsvxqxnhecpwyvwe0sf2qj4e.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Elrond NFT Swap: Marketplace",
+  "description": "Sell, Auction, Swap, Lend NFTs",
+  "tags": [
+    "krogan",
+    "marketplace"
+  ],
+  "icon": "krogan"
```

Comment:
for now, you can remove the `icon` tag and also the images from `icons` directory. They can be added later, if needed

Reply:
@bogdan-rosianu fixed ✅